docs/swarm_deployment.md

```diff
@@ -0,0 +1,56 @@
+**IMPORTANT** This deployment method is experimental, Docker Swarm experience is still required for volumes and container placement constraints!
```

Comment:
Sorry to be pedantic; can we please say it's "very" experimental? :smile: Just don't want anyone to try this early on and get frustrated by a snag.

s/experience/expertise